standards/JWT-SVID.md

```diff
@@ -0,0 +1,112 @@
+
+### 3.3. Expiration Time
+The `exp` claim MUST be set, and validators MUST reject tokens without this claim. Implementers are encouraged to keep the validity period as small as is reasonably possible, however this specification does not set any hard upper limits on its value.
+
+## 4. Token Signing and Validation
+JWT-SVID signing and validation semantics are the same as regular JWTs. Validators MUST ensure that the `alg` header is set to `RS256` before processing.
+
+JWT-SVID signatures are computed and validated following the steps outlined in [RFC 7515 section 7][2]. The `aud` and `exp` claims MUST be present and processed according to [RFC 7519][1] sections [4.1.3][3] and [4.1.4][4]. Validators receiving tokens without the `aud` and `exp` claims set MUST reject the token.
+
+## 5. Token Transmission
+This section describes the manner in which a JWT-SVID may be transmitted from one workload to another.
+
+### 5.1. Serialization
+JWT-SVIDs MUST be serialized using the JWS Compact Serialization method described in [RFC 7515 Section 3.1][5]. Note that this precludes the use of a JWS Unprotected Header, as mandated in the [JOSE Header](#2-jose-header) section.
+
+### 5.2. HTTP
+JWT-SVIDs transmitted via HTTP SHOULD be transmitted in the “Authorization” header (“authorization” for HTTP/2) using the “Bearer” authentication scheme defined in [RFC 6750 section 2.1][6]. For example, `Authorization: Bearer <serialized_token>` in HTTP/1.1 and `authorization: Bearer <serialized_token>` in HTTP/2.
+
+### 5.3. gRPC
+The gRPC protocol uses HTTP/2. As a result, the HTTP transmission guidelines in the [HTTP section](#52-http) equally apply. Concretely, gRPC implementations SHOULD set a metadata key `authorization` with a value of `Bearer <serialized_token>`.
+
+## 6. Security Considerations
+This section outlines the security considerations that implementers and users should take into account when using JWT-SVID.
+
+### 6.1. Replay Protection
+Being a bearer token, JWT-SVIDs are susceptible to replay attacks. By requiring that the `aud` and `exp` claims be set, this specification has taken steps to improve the situation, but is unable to solve it completely while retaining validation compatibility with [RFC 7515][1]. It is very important to understand this risk. Use of an aggressive value for the `exp` claim is recommended. Some users may wish to leverage the `jti` claim despite the added overhead. While use of the `jti` claim is permitted by this specification, it should be noted that JWT-SVID validators are not required to track `jti` uniqueness.
+
+### 6.2. Audience
+There is an implicit trust granted to recipients of JWT-SVIDs. Tokens sent to one audience can be replayed to another audience should more than one be present. As such, care should be taken when minting a JWT-SVID with more than one audience. Single audience JWT-SVID tokens are strongly recommended to limit the scope of replayability.
```

Comment:
> If there are two validators with different aud values, and the aud value is validated, then the tokens cannot be replayed between these two validators, because either the aud value will reject for the second validator, or the signature will fail for the second validator. (Or the attacker has the signing key in which case there are bigger problems.)

I'm not sure I follow this point... the key in use represents the subject, and not a relationship between the subject and the audience. Thus, the signature is valid for both audience values. If subject A cuts a token for audience X and Y, and sends that token to X, then X can replay the token to Y. Perhaps I am missing something?

Reply:
Ah, I misunderstood, I had two concerns, one of which is now addressed:
1. I thought you were worried about a token from sender A with (only) aud X being mistakenly accepted by aud Y.  (As you mentioned, this doesn't make that much sense for a token)
2. I also thought you were worried about a token from sender A with aud X being replayed _among members of audience X_.  I don't know how much of a concern this is, but if it is one, it's not addressed.

Assuming my second concern is a non-issue, I favor changing the text to clarify this for those who make the same misunderstanding:

> There is an implicit trust granted to recipients of JWT-SVIDs. A token from sender A to audiences X and Y can be replayed by audience X to audience Y.

Reply:
👍 I'll add the clarification - thanks for this